<launch>
  <arg name="input_objects" default="/perception/object_recognition/objects" />
  <arg name="input_path_topic" default="input/path" />
  <arg name="output_path_topic" default="output/path" />
  <arg name="is_showing_debug_info" default="false"/>
  <arg name="is_stopping_if_outside_drivable_area" default="true"/>
  <arg name="param_path" default="$(find-pkg-share obstacle_avoidance_planner)/config/default_obstacle_avoidance_planner.yaml" />

  <node pkg="obstacle_avoidance_planner" exec="obstacle_avoidance_planner_node"
    name="obstacle_avoidance_planner" output="screen">
    <remap from="input/objects" to="$(var input_objects)"/>
    <remap from="input/path" to="$(var input_path_topic)" />
    <remap from="output/path" to="$(var output_path_topic)" />

    <param from="$(var param_path)" />
    <param name="is_showing_debug_info" value="$(var is_showing_debug_info)"/>
    <param name="is_stopping_if_outside_drivable_area" value="$(var is_stopping_if_outside_drivable_area)"/>
  </node>
</launch>
